server/src/schemas/usersSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { gql } from "apollo-server-express";
+
+export const UsersTypeDefs = gql`
+  type User {
+    id: ID!
+    first_name: String!
+    last_name: String!
+    email: String!
+    type: Type!
```

Comment:
Can we rename this to something more specific, like `userType`? "type" is a protected word in a lot of cases.

Reply:
Removed as per is_student suggestion